slack-lint-checks/src/main/java/slack/lint/rx/RxObservableEmitDetector.kt

```diff
@@ -43,7 +43,7 @@ class RxObservableEmitDetector : Detector(), SourceCodeScanner {
                 sendCalled = true
               }
 
-              return super.visitCallExpression(node)
+              return false
```

Comment:
Weird...I just added this detector and it wasn't an issue. I'm guessing a recent lint update made this an issue